gcloud/resource_manager/test_client.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for resouce_manager.client.Client."""
+
+import unittest2
+
+
+class TestClient(unittest2.TestCase):
+    """Main test case for resource_manager.client.Client."""
+
+    def _getTargetClass(self):
+        from gcloud.resource_manager.client import Client
+        return Client
+
+    def _makeOne(self, *args, **kw):
+        return self._getTargetClass()(*args, **kw)
+
```

Comment:
I try to avoid brain dump commits for that very reason. It's way easier for the reviewer to have a tiny scope that can fit in the head at once. Can build the entire surface up over many commits / PRs.